packages/types/src/index.ts

```diff
@@ -249,14 +249,30 @@ export interface Feature {
   ranges?: Range[]; // if in a Group (mutex), these are the available slot ranges
 }
 
-export interface DatafileContent {
+export interface DatafileContentV1 {
```

Comment:
in `DatafileContentV1`, the entities were available in an array requiring an `O(n)` lookup.

with new `DatafileContentV2`, the lookup should be `O(1)`.